hera_cal/io.py

```diff
@@ -361,8 +370,9 @@ def build_datacontainers(self):
 
         return data, flags, nsamples
 
-    def read(self, bls=None, polarizations=None, times=None,
-             frequencies=None, freq_chans=None, read_data=True, return_data=True):
+    def read(self, bls=None, polarizations=None, times=None, frequencies=None,
+             freq_chans=None, read_data=True, return_data=True, run_check=True,
+             check_extra=True, run_check_acceptability=True):
```

Comment:
Same question as above, though this function is already pretty complicated so adding three more parameters is less of an issue.